### PR TITLE
Shin/ch1996/add tooltip info on add wallet page

### PR DESCRIPTION
**Pull request by sarveshwar-singh**
# Before
image

# After
image

image

image

image

### CODE REVIEW

File: app/components/wallet/WalletAdd.scss [hunk 1]
```diff
@@ -135,15 +143,6 @@
...
 
-    .heroCardsItem {
-      width: var(--card-width);
-      height: var(--card-height);
-      border-radius: var(--card-border-radius);
-      background-color: var(--card-bg-color);
-      color: var(--card-text-color);
-      font-size: var(--card-text-size);
-    }
-
```
Comment (sarveshwar-singh) [inline app/components/wallet/WalletAdd.scss]: these were redundent

File: app/components/wallet/WalletAdd.scss [hunk 2]
```diff
@@ -200,15 +199,6 @@
...
```
Comment (sarveshwar-singh) [inline app/components/wallet/WalletAdd.scss]: these were redundent

**Comment by SebastienGllmt** [comment]
@sarveshwar-singh merge conflict

**Comment by sarveshwar-singh** [comment]
@SebastienGllmt Fixed

**Pull request merged**